src/videoserver/lib/utils.py

```diff
@@ -14,6 +19,29 @@
 
 logger = logging.getLogger(__name__)
 
+s3 = boto3.client(
+    "s3",
+    aws_access_key_id=os.getenv('AWS_ACCESS_KEY'),
+    aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY')
```

Comment:
in general it would be better to use `app.config` for configuration rather than env directly,
so it's consistent with the rest